tests/test_blend.py

```diff
@@ -79,26 +72,21 @@ def test_init(self):
         assert blend.frame.psfs == frame.psfs
 
         # Test init with psfs
-        frame = scarlet.Frame(images.shape, psfs=target_psf[None], channels=channels)
-        obs1 = scarlet.Observation(images1, psfs=psfs1, channels=channels1).match(frame)
-        obs2 = scarlet.Observation(images2, psfs=psfs2, channels=channels2).match(frame)
-        observations = [obs1, obs2]
-        sources = [scarlet.PointSource(coord, frame, observations) for coord in coords]
+        frame = scarlet.Frame(images.shape, psfs=target_psf[None])
+        obs = scarlet.Observation(images, psfs=psfs).match(frame)
```

Comment:
The test is failing because you are using inconsistent names for the rest of this test. Here you define `obs`, but in some lines of code below you use `obs` and in others you use `observation`, which does not have a PSF and thus has different scaling.

Reply:
Even with fresh eyes this morning, I didn't see the mistake, but sure enough ...

Reply:
It does help to have written all of the tests so far... :)